docker-compose.base.yml

```diff
@@ -42,7 +42,7 @@ services:
                     }
 
                     handle @flags {
-                        reverse_proxy feature-flags:3001
+                        reverse_proxy  host.docker.internal:3001
```

Comment:
dark magic to get the proxy working for the flags service that's running locally.